website/docs/r/gcp_integration.html.markdown

```diff
@@ -35,7 +35,7 @@ resource "signalfx_gcp_integration" "gcp_myteam" {
 
 * `name` - (Required) Name of the integration.
 * `enabled` - (Required) Whether the integration is enabled.
-* `poll_rate` - (Required) GCP integration poll rate in seconds. Can be set to either 60 or 300 (1 minute or 5 minutes).
+* `poll_rate` - (Required) GCP integration poll rate (in seconds). Value between `60` and `600`.
```

Comment:
The other ones are optional and this one is required. Should we take this opportunity and unify that?